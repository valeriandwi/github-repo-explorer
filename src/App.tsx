import "./App.css";
import { RouterProvider, createBrowserRouter } from "react-router-dom";
import { PUBLIC_ROUTES } from "@/routes/routeList";

const router = createBrowserRouter(PUBLIC_ROUTES);

function App() {
  return <RouterProvider router={router} />;
}

export default App;
