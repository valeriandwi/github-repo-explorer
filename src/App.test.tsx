import { render } from "@testing-library/react";
import { describe, it, expect } from "vitest";
import App from "./App";

describe("App Component", () => {
  it("renders without crashing", () => {
    const { container } = render(<App />);
    expect(container).toBeDefined();
  });
});
