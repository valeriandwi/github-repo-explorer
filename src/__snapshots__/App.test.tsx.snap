// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App Component > matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex flex-row justify-center p-4"
  >
    <div
      class="ant-card ant-card-bordered lg:w-1/2 max-md:w-full md:w-3/4 css-dev-only-do-not-override-1wyde52"
    >
      <div
        class="ant-card-body"
      >
        <div
          class="flex flex-col space-y-4"
        >
          <div>
            <input
              class="ant-input ant-input-lg css-dev-only-do-not-override-1wyde52 ant-input-outlined !mb-4"
              data-testid="search-input"
              placeholder="Search Username..."
              type="text"
              value=""
            />
            <button
              class="ant-btn css-dev-only-do-not-override-1wyde52 ant-btn-primary ant-btn-color-primary ant-btn-variant-solid ant-btn-lg w-full bg-blue-200"
              data-testid="search-button"
              type="button"
            >
              <span>
                Search
              </span>
            </button>
          </div>
          <div />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
